init-odkx-sync-endpoint.py

```diff
@@ -17,12 +17,15 @@ def run_interactive_config():
     env_file_location = os.path.join(os.path.dirname(__file__), "config", "https.env")
 
     try:
-        domain, email = parse_env_file(env_file_location)
-        print("Found configuration at {}".format(env_file_location))
-    except OSError:
-        print("No default https configuration file found at expected path {}. This prevents automatically renewing certs!".format(env_file_location))
-        print("Please check your paths and file permissions, and make sure your config repo is up to date.")
-        exit(1)
+    domain, email, stack_name = parse_env_file(env_file_location)
```

Comment:
This looks good to me, except I think there is an indentation problem? (if that is fixed, the number of modified lines will also be reduced)